trufflehog/help.md

```diff
@@ -1,18 +1,15 @@
 # Description
 
-The [TruffleHog](https://github.com/dxa4481/truffleHog) plugin uses regex and entropy checks to go through the entire commit history of each branch, and check each diff for every commit as well as for secrets.
-
-For entropy checks, TruffleHog will evaluate the shannon entropy for both the base64 character set and hexadecimal character set for every blob of text greater than 20 characters comprised of those character sets in each diff.
-If at any point a high entropy string is greater than 20 characters is detected, it will collect the hashes and provide them for use once complete.
+Searche through git repositories for high entropy strings and secrets, digging deep into commit history
 
 # Key Features
 
-* Search through github commit histories 
-* Check for leaked secrets
+Identify key features of plugin.
```

Comment:
```suggestion
* Identify key features of plugin
```

Reply:
Actually, this also needs to be an actual feature of the plugin!